### PR TITLE
"Frontend UI for contract status widget"

### PR DESCRIPTION
**Pull request by richardcolachagua**
"frontend ui and logic written for contract status widget"

### CODE REVIEW

**Review by AlanisYates — COMMENTED** [review]
How does this look for you on your machine? also switch to using fake data now as the query does not exist but if you want to write the query function that would be in a seprate file.

File: src/components/Contract-Status-Widget.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+const GET_CONTRACT_STATUS_COUNTS = gql`
+  query GetContractStatusCounts {
+    contractStatusCounts {
+      notStarted
+      inProgress
+      done
+    }
+  }
+`;
```
Comment (AlanisYates) [inline src/components/Contract-Status-Widget.jsx:13]: This needs to be in a seperate file - see how `src/pages/SignupPage/signup.js` is done

File: src/context/apolloClient.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const httpLink = new HttpLink({
+  uri: "http://your-backend-uri/graphql",
+});
+
+const client = new ApolloClient({
+  link: httpLink,
+  cache: new InMemoryCache(),
+});
+
+export default client;
```
Comment (AlanisYates) [inline src/context/apolloClient.js:12]: This is not needed as its already in the index.js

Comment (richardcolachagua) [reply src/context/apolloClient.js:12]: removed

File: src/components/Contract-Status-Widget.jsx (hunk 1 above)
Comment (richardcolachagua) [reply src/components/Contract-Status-Widget.jsx]: turned into separate file

File: src/components/ContractStatusWidget/ContractStatusWidget.jsx [hunk 3]
```diff
@@ -14,30 +14,46 @@ export default function ContractStatusWidget() {
...
     >
-      <Typography variant="h1" component="div" gutterBottom>
-        {dummyData.contractStatusCounts.notStarted}
-      </Typography>
-      <Typography>Not Started</Typography>
+      <Box sx={{ textAlign: "center", mx: 2 }}>
+        <Typography variant="h1" component="div" gutterBottom>
+          {dummyData.contractStatusCounts.notStarted}
+        </Typography>
+        <Typography>Not Started</Typography>
```
Comment (AlanisYates) [inline src/components/ContractStatusWidget/ContractStatusWidget.jsx:41]: Bold this

File: src/components/ContractStatusWidget/ContractStatusWidget.jsx [hunk 4]
```diff
@@ -14,30 +14,46 @@ export default function ContractStatusWidget() {
...
 
-      <Typography variant="h1" component="div" gutterBottom>
-        {dummyData.contractStatusCounts.done}
-      </Typography>
-      <Typography>Done</Typography>
+      <Box sx={{ textAlign: "center", mx: 2 }}>
+        <Typography variant="h1" component="div" gutterBottom>
+          {dummyData.contractStatusCounts.done}
+        </Typography>
+        <Typography>Done</Typography>
```
Comment (AlanisYates) [inline src/components/ContractStatusWidget/ContractStatusWidget.jsx:48]: Bold this

File: src/components/ContractStatusWidget/ContractStatusWidget.jsx [hunk 5]
```diff
@@ -14,30 +14,46 @@ export default function ContractStatusWidget() {
...
 
-      <Typography variant="h1" component="div" gutterBottom>
-        {dummyData.contractStatusCounts.inProgress}
-      </Typography>
-      <Typography>In Progress</Typography>
+      <Box sx={{ textAlign: "center", mx: 2 }}>
+        <Typography variant="h1" component="div" gutterBottom>
+          {dummyData.contractStatusCounts.inProgress}
+        </Typography>
+        <Typography>In Progress</Typography>
```
Comment (AlanisYates) [inline src/components/ContractStatusWidget/ContractStatusWidget.jsx:55]: Bold This

**Review by AlanisYates — CHANGES_REQUESTED** [review]

File: src/components/ContractStatusWidget/ContractStatusWidget.jsx (hunk 3 above)
Comment (richardcolachagua) [reply src/components/ContractStatusWidget/ContractStatusWidget.jsx]: bolded

File: src/components/ContractStatusWidget/ContractStatusWidget.jsx (hunk 4 above)
Comment (richardcolachagua) [reply src/components/ContractStatusWidget/ContractStatusWidget.jsx]: bolded

File: src/components/ContractStatusWidget/ContractStatusWidget.jsx (hunk 5 above)
Comment (richardcolachagua) [reply src/components/ContractStatusWidget/ContractStatusWidget.jsx]: bolded

**Review by richardcolachagua — COMMENTED** [review]
bolded widget statuses

File: src/components/StripeWidget/StripeWidget.jsx
Comment (AlanisYates) [inline src/components/StripeWidget/StripeWidget.jsx:1]: Remove this file from this PR

**Review by AlanisYates — CHANGES_REQUESTED** [review]

**Pull request closed without merging**